File: prak3_java/Ram.java
public class Ram {
    
    private String capacity;
    private int price;

    //konstruktor
    public Ram() {
    }

    public Ram(String capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    //getter setter
    public String getCapacity() {
        return capacity;
    }

    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

}

File: prak3_c++/Ram.cpp
#include <string>

using std::string;

class Ram {
    private:
        string capacity;
        int price;
        
    public:
        //konstruktor
        Ram(){
        }

        Ram(string c, int p){
            capacity = c;
            price = p;
        }


        //setter
        void setCapacity(string c){
            capacity = c;
        }

        void setPrice(int p){
            price = p;
        }

        //getter
        string getCapacity(){
            return capacity;
        }

        int getPrice(){
            return price;
        }


        //destruktor
        ~Ram(){
        }

};
